### DISCUSSION TITLE
Cowrie removed from nextgen

### DISCUSSION THREAD

**Discussion by eschultze** [telekom-security/tpotce#1001] [General]
@t3chn0m4g3 just noticed Cowrie is being removed from nextgen, is that right? As of my understanding, endlessh is being added but it's not quite the same. I like the ability that Cowrie can collect malware samples. Any ideas on how to do that in the future? Thanks in advance!

**Comment by t3chn0m4g3** [comment]
The NextGen is and always has been our testing edition, of what might come. This does not mean that the next Standard edition will not be using Cowrie (of course not :bowtie:), but we will be introducing endlessh for a new edition.

**Reply by thaneye** [reply]
with cowrie we were catching commands input, I read that endlessh very slowly sends an endless, random SSH banner. Is there real value with it compared to cowrie?

**Reply by t3chn0m4g3** [reply]
As I mentioned, we will not be replacing Cowrie. If you wish to use Cowrie at the moment you can do so by running `tped.sh` and choose the `Standard` edition.
